src/SourceLink.Bitbucket.Git/GetSourceLinkUrl.cs

```diff
@@ -16,7 +16,101 @@ public sealed class GetSourceLinkUrl : GetSourceLinkUrlGitTask
         protected override string HostsItemGroupName => "SourceLinkBitbucketGitHost";
         protected override string ProviderDisplayName => "Bitbucket.Git";
 
+        private const string BitbucketCloudHostingUrl = "bitbucket.org";
+        private const string IsEnterpriseEditionMetadataName = "EnterpriseEdition";
+        private const string VersionMetadataName = "Version";
+        private const string VersionWithNewUrlFormat = "4.7";
+
+        protected override bool SupportsImplicitHost => false;
+
         protected override string BuildSourceLinkUrl(Uri contentUri, Uri gitUri, string relativeUrl, string revisionId, ITaskItem hostItem)
-            => UriUtilities.Combine(UriUtilities.Combine(contentUri.ToString(), relativeUrl), "raw/" + revisionId + "/*");
+        {
+            var isEnterpriseEditionFlagAvailable =
```

Comment:
>isEnterpriseEditionFlagAvailable [](start = 16, length = 32)

This can be simplified as 

```C#
return boo.TryParse(..., out var isEnterpriseEditionValue) && isEnterpriseEditionValue ?
            BuildSourceLinkUrlForEnterpriseEdition(contentUri, relativeUrl, revisionId, hostItem) : 
            BuildSourceLinkUrlForCloudEdition(contentUri, relativeUrl, revisionId); 
```

Reply:
@tmat I modified pull request according to your suggestions.
- Now the `SourceLinkBitbucketGitHost` for bitbucket.org in props have `EnterpriseEdition="false"`
- Removed `SupportsImplicitHost => false;`
- I had to check if `hostItem` is null. It is null if `SourceLinkBitbucketGitHost ` does not match actual git url host.

Bottom line, now it will be not necessary to put  `SourceLinkBitbucketGitHost` in csproj(unless you want to set specifiv Version attribute) and it will work for both cloud and enterprise scenarios.